test/test-debuglet.ts

```diff
@@ -77,6 +77,41 @@ function verifyBreakpointRejection(re: RegExp, body: {breakpoint: any}) {
   return status.isError && hasCorrectDescription;
 }
 
+describe('CachedPromise', function() {
+  it('CachedPromise.get() is initially null', (done) => {
+    let cachedPromise = new CachedPromise();
+    assert(cachedPromise.get() === null);
+    done();
+  });
+
+  it('CachedPromise.get() is not null after refresh', (done) => {
+    let cachedPromise = new CachedPromise();
+    cachedPromise.refresh();
+    assert(cachedPromise.get() !== null)
+    done();
+  });
+
+  it('CachedPromise.get() will return a promise resolved after CachedPromise.resolve() is called', (done) => {
+    this.timeout(2000);
+    let cachedPromise = new CachedPromise();
+    cachedPromise.refresh();
+    (cachedPromise.get() as Promise<void>).then(() => {
+      done();
+    });
+    cachedPromise.resolve();
+  });
+  it('CachedPromise.get() will be null if it has been cleared', (done) => {
+    this.timeout(2000);
+    let cachedPromise = new CachedPromise();
+    assert(cachedPromise.get() === null);
```

Comment:
I would use `assert.stringEqual(cachedPromise.get(), null)` here.  That way if the assert fails, the actual value will be printed with the expected value.  Otherwise it will just print that `false` was encountered but `true` was expected.

Reply:
Done.